training/train_yolo.py

```
@@ -1,72 +1,161 @@
+def train_yolo_model(training_config: TrainingConfig) -> None:
+    """Train YOLO model based on the provided configuration.
+
+    Args:
+        training_config (TrainingConfig): Training configuration.
+    """
+
+    # Load pre-trained YOLOv11 model. The model is downloaded from Ultralytics.
+    model = YOLO(training_config.model)
+
+    # Train the model
     results = model.train(
```

Ruff (0.14.4)

Local variable `results` is assigned to but never used

Remove assignment to unused variable `results`

(F841)